src/main/scala/io/igl/jwt/DecodedJwt.scala

```diff
@@ -117,39 +122,60 @@ object DecodedJwt {
-   * Including an algorithm field in the requiredHeaders set is not needed, instead use the requiredAlg parameter.
-   *
-   * @param jwt an encrypted jwt
-   * @param key the key to use when validating the signature
-   * @param requiredAlg the algorithm to require and use when validating the signature
-   * @param requiredHeaders the headers the encrypted jwt is required to use
-   * @param requiredClaims the claims the encrypted jwt is required to use
-   * @param ignoredHeaders the headers to ignore should the encrypted jwt use them
-   * @param ignoredClaims the claims to ignore should the encrypted jwt use them
-   * @param iss used optionally, when you want to only validate a jwt where its required iss claim is equal to this
-   * @param aud used optionally, when you want to only validate a jwt where its required aud claim is equal to this
-   * @param iat used optionally, when you want to only validate a jwt where its required iat claim is equal to this
-   * @param sub used optionally, when you want to only validate a jwt where its required sub claim is equal to this
-   * @param jti used optionally, when you want to only validate a jwt where its required jti claim is equal to this
-   * @return returns a [[DecodedJwt]] wrapped in Success when successful, otherwise Failure
-   */
+    * This method use the underlying method {@link #validateEncodedJwtWithEncodedSecret(String,Array[Byte],Algorithm,Set[HeaderField],Set[ClaimField],Set[String],Set[String],Option[Iss],Option[Aud], Option[Iat], Option[Sub],Option[Jti],String)},
+    * by providing the secret with {@link String#getBytes(StandardCharsets#UTF_8)}
+    */
   def validateEncodedJwt(
-    jwt: String,
-    key: String,
-    requiredAlg: Algorithm,
-    requiredHeaders: Set[HeaderField],
-    requiredClaims: Set[ClaimField],
-    ignoredHeaders: Set[String] = Set(),
-    ignoredClaims: Set[String] = Set(),
-    iss: Option[Iss] = None,
-    aud: Option[Aud] = None,
-    iat: Option[Iat] = None,
-    sub: Option[Sub] = None,
-    jti: Option[Jti] = None,
-    charset: String = "UTF-8"): Try[Jwt] = Try {
+                          jwt: String,
+                          key: String,
+                          requiredAlg: Algorithm,
+                          requiredHeaders: Set[HeaderField],
+                          requiredClaims: Set[ClaimField],
+                          ignoredHeaders: Set[String] = Set(),
+                          ignoredClaims: Set[String] = Set(),
+                          iss: Option[Iss] = None,
+                          aud: Option[Aud] = None,
+                          iat: Option[Iat] = None,
+                          sub: Option[Sub] = None,
+                          jti: Option[Jti] = None,
+                          charset: String = "UTF-8"): Try[Jwt] = {
+    validateEncodedJwtWithEncodedSecret(jwt, key.getBytes(UTF_8), requiredAlg, requiredHeaders, requiredClaims, ignoredHeaders, ignoredClaims, iss, aud, iat, sub, jti, charset)
```

Comment:
This line is pretty long, perhaps put each parameter on it's own line? Similar to how the constructor parameters are written above.